pkgs/applications/video/obs-studio/plugins/obs-multi-rtmp/default.nix

```diff
@@ -11,6 +11,8 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-MRBQY9m6rj8HVdn58mK/Vh07FSm0EglRUaP20P3FFO4=";
   };
 
+  patches = [ ./fix-qt.patch ];
```

Comment:
Please reference upstream patch by commit using `fetchpatch`.

Like this:
https://github.com/sorayuki/obs-multi-rtmp/commit/a1289fdef404b08a7acbbf0d6d0f93da4c9fc087.patch

Reply:
To clarify, this is what we like to do for merged commits (which this is)

Doing it on in-review commits off a pr is disliked as the commit can disappear when the branch is deleted or force pushed over in review. In that case committing the patch is preferred.

---

If you could also add a comment saying to remove it next release that'd be perfect. Thanks Jack

Reply:
It looks like there are no cleanly-applying patches available (seemingly due to this merge: https://github.com/sorayuki/obs-multi-rtmp/commit/33b33a11cb03b74d5c77e0434b440957121b3b0b), so I've had to use `includes` to just patch `CMakeLists.txt` (the patch was failing on `CMakePresets.json`, which conflicted in that merge).